internal/db/db.go

```diff
@@ -0,0 +1,40 @@
+package db
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+)
+
+type Database struct {
+	Client *sqlx.DB
+}
+
+func NewDatabase() (*Database, error) {
+	connectionString := fmt.Sprintf(
+		"host=%s port=%s user=%s dbname=%s password=%s sslmode=%s",
+		os.Getenv("DB_HOST"),
+		os.Getenv("DB_PORT"),
+		os.Getenv("DB_USERNAME"),
+		os.Getenv("DB_TABLE"),
+		os.Getenv("DB_PASSWORD"),
+		os.Getenv("SSL_MODE"),
+	)
```

Comment:
connecting to the api - build - environment variables setup in the docker-compose.yml file